<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TeachDock</class>
 <widget class="QWidget" name="TeachDock">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>259</width>
    <height>321</height>
   </rect>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="tabletTracking">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>Teach</string>
  </property>
  <widget class="QPushButton" name="pushButtonAlign">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>171</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Align</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonLoadCSV">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>101</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>load CSV</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxAlignMode">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>142</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMove">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>23</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMoveNext">
   <property name="geometry">
    <rect>
     <x>73</x>
     <y>23</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Next Move</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxFindAndMove">
   <property name="geometry">
    <rect>
     <x>4</x>
     <y>63</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Find &amp;&amp; Move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonMoveScrew">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>24</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Screw Move</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>241</width>
     <height>17</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(222, 222, 230);</string>
   </property>
   <property name="text">
    <string>Feducial</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVisionFeducial">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>230</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vision</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditVisionFeducialPos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>233</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonVisionFeducialMove">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>230</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonScrewFeducial">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>260</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Screw</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditScrewFeducialPos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>261</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonScrewFeducialMove">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>260</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Move</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFeducialOffset">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>290</y>
     <width>81</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>End/Offset</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditFeducialOffset">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>83</x>
     <y>291</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>true</italic>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonFeducialOffsetSave">
   <property name="geometry">
    <rect>
     <x>188</x>
     <y>290</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonSaveScrewPos">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>144</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Save
Screw
Position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonReloadScrewPos">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>144</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Reload
Screw
Position</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonHoleFind">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>101</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hole Find</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAutoTeach">
   <property name="geometry">
    <rect>
     <x>84</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Auto Teach</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAF">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>63</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Ubuntu</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>AF</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAutoFocus">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>63</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
    </font>
   </property>
   <property name="text">
    <string>Auto Focus</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
